aws/rust-runtime/aws-config/src/profile/mod.rs

```diff
@@ -16,7 +16,7 @@ mod parser;
 #[doc(inline)]
 pub use parser::ProfileParseError;
 #[doc(inline)]
-pub use parser::{load, Profile, ProfileFileLoadError, ProfileSet, Property};
+pub use parser::{load, Profile, ProfileFileLoadError, ProfileSet, Property, SsoSession};
```

Comment:
should carefully consider if we want to make this public. I don't think it's required to be right now.

Reply:
Fixed in 3ab8cfac3748f7e42e1707abe80e5425ffa0ba95